Sources/Steepshot/Steepshot.Core/HttpClient/SteemClient.cs

```diff
@@ -420,6 +395,10 @@ public override bool TryReconnectChain(CancellationToken token)
                     Metadata = JsonConverter.Deserialize<AccountMetadata>(acc.JsonMetadata)
                 };
 
+                var balances = result.Result.Balances = new Dictionary<CurrencyType, (long Value, byte precision, string ChainCurrency)>();
```

Comment:
It seems that the balances is not used anywhere. Try to use  result.Result.Balances = new Dictionary<CurrencyType,*Model>(){{key,value},{key,value}}